konrad/core.py

```diff
@@ -350,6 +375,32 @@ def run(self):
 
             # Calculate temperature change for convergence check.
             self.deltaT = (self.atmosphere['T'] - T) / self.timestep_days
+            
+            # Calculate the delta for the radiative flux at the TOA.
+            self.delta2 = (4*constants.stefan_boltzmann*
+                           (self.atmosphere['T'][0,-1]**3)*
+                           self.delta
+                          )
```

Comment:
I am not sure, if I understand your convergence criterion. I though that you want the absolute change in TOA to be smaller than a given delta. But here, you check if it is smaller than the temperature-derivative of black-body radiation.
Could you elaborate on this decision? We can also meet offline and discuss it ;)